HSAttachmentPicker/Classes/HSAttachmentPicker.m

```diff
@@ -6,11 +6,15 @@
 #import "HSAttachmentPickerPhotoPreviewController.h"
 
 @interface HSAttachmentPicker () <HSAttachmentPickerPhotoPreviewControllerDelegate, UIDocumentMenuDelegate, UIDocumentPickerDelegate, UIImagePickerControllerDelegate, UINavigationControllerDelegate>
+
+@property(nonatomic) HSAttachmentPicker *selfReference;
```

Comment:
Interesting, is this just to keep the attachment picker alive which it's presented so that nobody has to hold onto a reference to it?